From error27@gmail.com  Wed Jan 27 14:35:52 2010
From: Dan Carpenter <error27@gmail.com>
Date: Mon, 18 Jan 2010 14:23:20 +0300
Subject: Staging: rtl8187se: fix if statement
To: Greg Kroah-Hartman <gregkh@suse.de>
Message-ID: <20100118112319.GB31923@bicker>


I removed a misplace semicolon.  It is clear from the indentation that
TxPwrTracking87SE() was only supposed to be called if CheckTxPwrTracking()
returned true.

Signed-off-by: Dan Carpenter <error27@gmail.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/rtl8187se/r8180_core.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/staging/rtl8187se/r8180_core.c
+++ b/drivers/staging/rtl8187se/r8180_core.c
@@ -2566,7 +2566,7 @@ void watch_dog_adaptive(unsigned long da
 	// Tx Power Tracking on 87SE.
 #ifdef TX_TRACK
 	//if( priv->bTxPowerTrack )	//lzm mod 080826
-	if( CheckTxPwrTracking((struct net_device *)data));
+	if (CheckTxPwrTracking((struct net_device *)data))
 		TxPwrTracking87SE((struct net_device *)data);
 #endif
 
